inductiva/tasks/run_simulation.py

```diff
@@ -65,6 +65,11 @@ def run_simulation(
         raise RuntimeError(
             f"Expected result to be a string with task_id, got {type(task_id)}")
 
+    position = task.get_position_in_queue()
+    if position is not None:
+        logging.info("There are %s tasks ahead of this task in the queue.",
```

Comment:
Minor detail: the plural does not quite work for a single task... it should be "there is 1 task" and not "there are 1 tasks"... If you want to avoid an if-statement just for this, maybe say "Number of tasks ahead in the queue: %s"... (or something along those lines)

But this is great feedback for the user, thanks Francisco!